hedera-mirror-rest/__tests__/globalTeardown.js

```diff
@@ -15,15 +15,19 @@
  */
 
 import fs from 'fs';
-import {getDatabases} from './globalSetup';
 import log4js from 'log4js';
+import {getContainerRuntimeClient} from 'testcontainers';
 
 export default async function () {
-  const tmpDir = process.env.MIGRATION_TMP_DIR;
-  if (tmpDir) {
-    fs.rmSync(tmpDir, {force: true, recursive: true});
-    console.log(`Removed temp directory ${tmpDir}`);
+  const containerIds = fs.readdirSync(process.env.CONTAINER_TEMP_DIR);
+  const client = await getContainerRuntimeClient();
+  for (const id of containerIds) {
+    const container = client.container.getById(id);
+    await client.container.stop(container);
+    await client.container.remove(container);
```

Comment:
Instead of creating a temp dir, writing the container IDs there, then reading those back to remove we could reuse the existing state in docker. Can just generate a global string like `mirror-rest: ${parentProcessId}` and attach it as a label to each container when starting. `parentProcessId` can be the actual parent process ID or some random integer generated by the parent. There doesn't seem to be a way to list by label, but we can do multiple `fetchByLabel()` until it deletes them all.

```suggestion
  const client = await getContainerRuntimeClient();
  while (true) {
    const container = client.container.fetchByLabel("mirror-rest", parentProcessId);
    if (!container) {
      break;
    }
    await client.container.stop(container);
    await client.container.remove(container);
```

Reply:
Sounds good, I have implemented this.